modin/pandas/test/test_io.py

```diff
@@ -335,7 +335,6 @@ def test_read_csv_parsing_1(
 
         eval_io(
             fn_name="read_csv",
-            check_exception_type=None,  # issue #2320
```

Comment:
Why don't we want to check exception type anymore?

Reply:
On the contrary, now we are checking. `None` was used to disable validation.

Reply:
Sorry for confusion 😄